config/delete_persisted_logs_crontab

```diff
@@ -0,0 +1,3 @@
+# Recurring persist_log garbage collection function handled by cron
+# Installed by AmpliPi
+0 0 * * * /usr/local/bin/delete_persisted_logs.sh
```

Comment:
I would prefer we don't implement this. We already set `SystemMaxUse` to 64mb. journalctl does not need log rotation, it will remove the oldest entries first when it gets to 64mb.

Reply:
The problem being solved here is that once you stop persisting new logs, the logs you've already persisted never get wiped and will still be at the top of amplipi.local:19531/entrie.
I perceive that to be an issue and so made this cronjob that fires off a bash script, the bash script uses curl to call the `DELETE /settings/logs` endpoint, this endpoint conditionally clears out persisted logs if storage is set to volatile (aka if persistence is off)

Reply:
1) if the max filesize of logs is only ever 64mb, is that actually necessary? The reason why logs aren't saved to disk right now is to prevent excessive writes, not because the size on disk is excessive. I'm unconvinced of its necessity, and tend to think instead this does actions beyond what's "written on the can", which could be problematic.
2) if you wanted to clean the logs when toggling this, why would you choose a cronjob to do that rather than the code that's actually doing the toggling? if you're worried about the call taking too long, fork it to the background
3) the entries logs being at the top is definitely something for us to be aware of when we are getting & viewing logs. however, if someone has chosen to persist logs, would you not expect them to ... persist? they chose that for a reason, probably because they were having a problem.

imo we should probably persist the logs users have asked us to persist, or make it abundantly clear that toggling "persist logs" off will wipe any persisted logs.

Reply:
Path of least resistance says I leave it as it is in the current version by not deleting logs, but I'll still answer questions for the context:

1. Maybe, only if you care about not needing to search for up to date logs when you looks at the logs page
2. Cronjob was just in case the logs were accidentally toggled back, or if someone didn't realize that turning that off would un-persist logs before they got the chance to collect them
3. Fair enough